packages/rum/test/specs/apm-base.spec.js

```diff
@@ -89,12 +92,11 @@ describe('ApmBase', function() {
     req.send()
     const tr = apmBase.getCurrentTransaction()
     expect(tr).toBeUndefined()
-    expect(loggingService.info).toHaveBeenCalledWith('RUM agent is inactive')
```

Comment:
Isn't this more clear compared to moving it to the `callFake`?

Reply:
It didn't work for me in other tests, so I changed it to keep it consistent.